client/src/featureform/register.py

```diff
@@ -2132,13 +2356,12 @@ def register_blob_store(
         root_path: str,
         description: str = "",
         team: str = "",
-        tags: List[str] = [],
-        properties: dict = {},
+        tags=None,
+        properties=None,
     ):
-        """Register an azure blob store provider.
+        """Register an Azure Blob Store provider.
 
-        This has the functionality of an online store and can be used as a parameter
-        to a k8s or spark provider
+        Azure Blob Storage can be used as the storage component for Spark or the Featureform Pandas Runner.
```

Comment:
for the example below, can we make `account_name` and `account_key` into string?